scripts/build_docs.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
```

Comment:
One thing that stands out to me are the python scripts here. Since we're already in node world with Docusaurus and all, it would feel natural to me to have node scripts instead? That would likely also integrate much nicer with the package scripts.

Reply:
Thanks for taking a look, Andy! They're Python scripts mostly cause that's what @vtlim and I are most comfortable with. We don't have a strong attachment to them if anyone wants to redo them in a different language.

Reply:
Cool! I thought that might be the case, which makes sense; maybe @jgoz or myself will find some time. If not, we could always update this later on.